contributors/design-proposals/dir_struct.txt

```diff
@@ -0,0 +1,240 @@
+Uncategorized (Please Help)
+	high-availability.md
+	control-plane-resilience.md
+	downward_api_resources_limits_requests.md
+	seccomp.md
+	client-package-structure.md
+	service-discovery.md
+	metadata-policy.md
+	containerized-mounter.md~
+	identifiers.md
+	local-cluster-ux.md
+	pod-pid-namespace.md
+	grow-volume-size.md
+	image-provenance.md
+	core-metrics-pipeline.md
+	versioning.md
+	ha_master.md
+	secret-configmap-downwarapi-file-mode.md
+	protobuf.md
+	flakiness-sla.md
+	resources.md
+	initial-resources.md
+	create_sheet.py
+	runtime-client-server.md
+	OWNERS
+	namespaces.md
+	cpu-manager.md
```

Comment:
It's pretty inconvenient that these aren't alphabetical.

Node, or Resource Management WG